ssl/statem/statem_dtls.c

```diff
@@ -16,9 +16,6 @@
 #include "internal/cryptlib.h"
 #include "internal/ssl_unwrap.h"
 #include <openssl/buffer.h>
-#include <openssl/objects.h>
-#include <openssl/evp.h>
-#include <openssl/x509.h>
```

Comment:
These deletions seem unrelated. Although I guess if the build is happy without them, I suppose its ok.